CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 - New config values `password_reset_token_lifetime`
   and `email_verification_token_lifetime`: https://github.com/supertokens/supertokens-core/issues/297
+- Added support for multiple access token signing keys: https://github.com/supertokens/supertokens-core/issues/305
```

Comment:
Update the changelog to specify db schema changes

Reply:
In your message for this, please state the name of the table that was added.

Reply:
Should I add the table name here? It feels a bit out of place since it's different in Mongo.

Reply:
Just say something like "Added a new table named ... for SQL storage"